### PR TITLE
Features/login

### PR DESCRIPTION
**Pull request by JkGemma**
Connexion & Gestion de Session (localstorage stockage du refreshToken & du token)

### CODE REVIEW

File: pages/login.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    <TextField size="long" thickness="large" error={badUser} onChange={(e) => {handlePassword(e)}} type="password" placeholder="********" />
+                    <Button size="long" variant="teal" thickness="large">Login</Button>
+                </form>
+            {badUser && <Text variant="h5" align="left" color={theme.cvar('colorSelectTextError')}>{errorMsg}</Text> }
+            </Container>
+        </Container>
+    );
+};
+
+export default Login;
```
Comment (PixDay) [inline pages/login.tsx]:
```suggestion
export default Login;

```

**Comment by shellbear** [comment]
#### 📝Changed routes:


| [**/login**](https://bird-2qxbkn60g-hiddentity.vercel.app/login) |
|:-:|
|<a href="https://bird-2qxbkn60g-hiddentity.vercel.app/login">Screenshot of /login</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/S9l50UsuETBIlnSFAHP4jN9FuGbWhcLrGNdNC3NhVfi_mHKbUmIS-YVg8hYmmQ3Hvy431pqjqpeEDTSSNY61CQ.png">(view full size)</a> |







Commit c41ff4e7aea8b05ed7af2d6ec00321b211d1157b (https://bird-2qxbkn60g-hiddentity.vercel.app).

File: pages/login.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import React from 'react';
```
Comment (ysainson) [inline pages/login.tsx]:
```suggestion
import * as React from 'react';
```

File: pages/login.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        Router.push("/");
+        // return (
+            // <Container align="center" gap={0}>You are logged in.</Container>
+            // redirect to main page, token saved on localstorage, possibility to get a cookie if needed
+            // )
+    }
+    return (
+        <Container align="center" gap={0}>
+            <Container row align="center">
+                <img width={50} height={50} src="/images/logo-color.png" alt="hiddentity logo" className="hiddentity-logo" />
```
Comment (ysainson) [inline pages/login.tsx]:
```suggestion
                <Logo />
```

File: pages/login.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                <Text variant="h3" color={theme.cvar('colorForeground')}>Hiddentity</Text>
+            </Container>
+            <Container gap={0}>
+                <form onSubmit={e => {e.preventDefault(); logger() }} key="form">
```
Comment (ysainson) [inline pages/login.tsx]: Pas besoin d'utiliser `form`, juste des `TextField` et un `Button`

**Review by ysainson — CHANGES_REQUESTED** [review]

File: pages/login.tsx (hunk 4 above)
Comment (JkGemma) [reply pages/login.tsx]: Done in last commit.

**Review by ysainson — APPROVED** [review]

**Pull request merged**